From 4b92037ef782acbb3e1252385dbe1dc5aeed4292 Mon Sep 17 00:00:00 2001
From: Sriram Nambakam <snambakam@microsoft.com>
Date: Tue, 17 Mar 2026 16:32:13 -0700
Subject: [PATCH 32/49] Fix compile error

---
 arch/x86/kvm/vmx/vmx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/kvm/vmx/vmx.c b/arch/x86/kvm/vmx/vmx.c
index 39cb0945f686..6b3cb1eb494a 100644
--- a/arch/x86/kvm/vmx/vmx.c
+++ b/arch/x86/kvm/vmx/vmx.c
@@ -409,7 +409,7 @@ static noinstr void vmx_l1d_flush(struct kvm_vcpu *vcpu)
 		kvm_clear_cpu_l1tf_flush_l1d();
 	}
 
-	vcpu->stat.l1d_flush++;
+	vcpu->stat->l1d_flush++;
 
 	if (static_cpu_has(X86_FEATURE_FLUSH_L1D)) {
 		native_wrmsrq(MSR_IA32_FLUSH_CMD, L1D_FLUSH);
-- 
2.53.0

